openscm/adapters/dice.py

```diff
@@ -0,0 +1,388 @@
+"""
+Adapter for the climate component from the Dynamic Integrated Climate-Economy (DICE)
+model by William Nordhaus.
+
+Original source: https://sites.google.com/site/williamdnordhaus/dice-rice
+"""
+
+from collections import namedtuple
+from datetime import datetime
+from math import log2
+from typing import Any
+
+import numpy as np
+
+from ..adapter import Adapter
+from ..parameters import ParameterType
+from ..timeseries_converter import (
+    ExtrapolationType,
+    InterpolationType,
+    create_time_points,
+)
+
+YEAR = 365 * 24 * 60 * 60
+
+MODEL_PARAMETER_DEFAULTS = {
+    # Initial concentration in atmosphere
```

Comment:
```suggestion
    # Initial size of carbon pool in atmosphere (referred to as concentration in original publication but units show this is wrong
```

Reply:
same for all the pools below which have units of "GtC"

Reply:
Ok. Will rename the comments and parameters together with a PR to #139